crates/holochain/src/core/workflow/app_validation_workflow/network_call_tests.rs

```diff
@@ -120,6 +124,168 @@ async fn get_validation_package_test() {
     ConductorTestData::shutdown_conductor(handle).await;
 }
 
+#[tokio::test(threaded_scheduler)]
+async fn get_agent_activity_test() {
+    observability::test_run().ok();
+
+    // Check if the correct number of ops are integrated
+    // every 100 ms for a maximum of 10 seconds but early exit
+    // if they are there.
+    let num_attempts = 100;
+    let delay_per_attempt = std::time::Duration::from_millis(100);
```

Comment:
```suggestion
    const NUM_ATTEMPTS: usize = 100;
    const DELAY_PER_ATTEMPT_MS = 100;
    
    let delay_per_attempt = std::time::Duration::from_millis(DELAY_PER_ATTEMPT_MS);
```

The all caps helps me know these things could be tweak-able if the test ends up being flaky : )